Office 365/o365/ingest/parser.yml

```
@@ -221,8 +296,9 @@ stages:
           event.category: ["file"]
           event.type: ["info"]
           user_agent.original: "{{json_event.message.UserAgent}}"
+          file.extension: "{{json_event.message.SourceFileExtension}}"
           action.outcome: "success"
-          url.full: "{{json_event.message.ObjectId}}"
+          url.full: "{{json_event.message.SiteUrl}}"
```

According to some tests, we lost information. I suggest:
```suggestion
          url.full: "{{json_event.message.ObjectId or json_event.message.SiteUrl}}"
```